integreat_cms/cms/templates/_base.html

```diff
@@ -105,33 +105,39 @@
                     {% trans "My Dashboard" %}
                 </a>
                 <!-- Analytics Section -->
-                <div class="min-h-3 pl-2">
-                    <p class="font-bold">{% trans "Analytics" %}</p>
-                </div>
-                {% if request.region.statistics_enabled %}
-                    <a href="{% url 'statistics' region_slug=request.region.slug %}"
-                       class="{% if current_menu_item == 'statistics' %} active{% endif %}">
-                        <i icon-name="trending-up"></i>
-                        {% trans "Statistics" %}
-                    </a>
-                {% endif %}
-                <a href="{% url 'translation_coverage' region_slug=request.region.slug %}"
-                   class="{% if current_menu_item == 'translation_coverage' %} active{% endif %}">
-                    <i icon-name="check-circle"></i>
-                    {% trans "Translation Report" %}
-                </a>
-                {% if perms.cms.view_feedback %}
-                    <a href="{% url 'region_feedback' region_slug=request.region.slug %}"
-                       class="{% if current_menu_item == 'region_feedback' %} active{% endif %}">
-                        <i icon-name="thumbs-up"></i>
-                        {% trans "Feedback" %}
-                    </a>
+                {% if perms.cms.view_analytics %}
```

Comment:
```suggestion
                {% if request.region.statistics_enabled and perms.cms.view_statistics or perms.cms.view_translation_report or perms.cms.view_feedback or perms.cms.view_broken_links %}
```
I don't think the `view_analytics` permission makes sense on its own - we should instead just show the analytics section whenever the user is allowed to see at least one part of it.

Reply:
Done.